resource-managers/kubernetes/core/src/test/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorRollPluginSuite.scala

```diff
@@ -172,10 +174,53 @@ class ExecutorRollPluginSuite extends SparkFunSuite with PrivateMethodTester {
       plugin.invokePrivate(_choose(list, ExecutorRollPolicy.AVERAGE_DURATION)))
   }
 
-  test("Policy: OUTLIER - Work like TOTAL_DURATION if there is no outlier") {
+  test("Policy: OUTLIER_OR_TOTAL_DURATION - Use TOTAL_DURATION if there is no outlier") {
     assertEquals(
       plugin.invokePrivate(_choose(list, ExecutorRollPolicy.TOTAL_DURATION)),
-      plugin.invokePrivate(_choose(list, ExecutorRollPolicy.OUTLIER)))
+      plugin.invokePrivate(_choose(list, ExecutorRollPolicy.OUTLIER_OR_TOTAL_DURATION)))
+  }
+
+  test("Policy: OUTLIER_OR_TOTAL_DURATION - Detect an average task duration outlier") {
```

Comment:
Actually, this is orthogonal to your proposal. We usually put one theme in a PR and avoid mixing different things.
Could you make a separate PR for adding more test coverage for `OUTLIER_OR_TOTAL_DURATION`?